src/runtime-rs/Makefile

```diff
@@ -49,9 +48,7 @@ DEFAULT_HYPERVISOR ?= $(HYPERVISOR_DB)
 
 HYPERVISORS := $(HYPERVISOR_DB)
 
-DBPATH := $(DBBINDIR)/$(QEMUCMD)
-DBVALIDHYPERVISORPATHS := [\"$(DBPATH)\"]
-
+DBVALIDHYPERVISORPATHS := []
```

Comment:
Is `DBVALIDHYPERVISORPATHS` itself still needed?

Reply:
yes, because runtime will check the value of valid_hypervisors, for Dragonball case, we need it to be empty.